packages/contracts/docs/core/01-how-it-works/01-core/02-permissions/index.md

```diff
@@ -0,0 +1,153 @@
+
+Granting a permission with `_where: ANY_ADDR` to a condition has the effect that is granted on every contract. This is useful if you want to give an address `_who` permission over a large set of contracts that would be too costly or too much work to be granted on a per-contract basis.
+Imagine, for example, that many instances of the `Service` contract exist, and a user should have the permission to use all of them. By granting the `USE_PERMISSION_ID` with `_where: ANY_ADDR`, to some user `_who: userAddr`, the user has access to all of them. If this should not be possible anymore, you can later revoke the permission.
+
+However, some restrictions apply. For security reasons, aragonOS does not allow you to use both, `_where: ANY_ADDR` and `_who: ANY_ADDR` in the same permission. Furthermore, the permission IDs of [permissions native to the `DAO` Contract](#permissions-native-to-the-dao-contract) cannot be used.
+
+### Permissions Native to the `DAO` Contract
+
+The following functions in the DAO are permissioned:
+
+| Functions                               | Permission Identifier                      | Description                                                                                                     |
+| --------------------------------------- | ------------------------------------------ | --------------------------------------------------------------------------------------------------------------- |
+| `grant`, `grantWithCondition`, `revoke` | `ROOT_PERMISSION_ID`                       | Required to manage permissions of the DAO and associated plugins.                                               |
+| `execute`                               | `EXECUTE_PERMISSION_ID`                    | Required to execute arbitrary actions.                                                                          |
+| `_authorizeUpgrade`                     | `UPGRADE_DAO_PERMISSION_ID`                | Required to upgrade the DAO (via the [UUPS](https://eips.ethereum.org/EIPS/eip-1822)).                          |
+| `setMetadata`                           | `SET_METADATA_PERMISSION_ID`               | Required to set the DAO’s metadata and [DAOstar.one DAO URI](https://eips.ethereum.org/EIPS/eip-4824).          |
+| `setTrustedForwarder`                   | `SET_TRUSTED_FORWARDER_PERMISSION_ID`      | Required to set the DAO’s trusted forwarder for meta transactions.                                              |
+| `setSignatureValidator`                 | `SET_SIGNATURE_VALIDATOR_PERMISSION_ID`    | Required to set the DAO’s signature validator contract (see ERC-1271).                                          |
+| `registerStandardCallback`              | `REGISTER_STANDARD_CALLBACK_PERMISSION_ID` | Required to register a standard callback for an [ERC-165](https://eips.ethereum.org/EIPS/eip-165) interface ID. |
+
+Plugins installed on the DAO might introduce other permissions and associated permission identifiers.
```

Comment:
I would be a bit more verbose here. I would say that permission ID's are completely open to anybody to create their own. It's an arbitrary value that any developer (of a plugin or not) could define and grant at will.

Reply:
Agreed, but not in the section titled `### Permissions Native to the `DAO` Contract`.
Can we bundle this with the task for https://github.com/aragon/core/pull/247#discussion_r1097617341 ?

Reply:
I have put it into the task description of https://aragonassociation.atlassian.net/browse/APP-1732